treeherder/services/pulse/consumers.py

```diff
@@ -45,14 +46,15 @@ class PulseConsumer(ConsumerMixin):
     Consume jobs from Pulse exchanges
     """
 
-    def __init__(self, source, build_routing_key):
+    def __init__(self, source, build_routing_key, options={}):
         self.connection = Connection(source['pulse_url'])
         self.consumers = []
         self.queue = None
         self.queue_name = "queue/{}/{}".format(self.connection.userid, self.queue_suffix)
         self.root_url = source['root_url']
         self.source = source
         self.build_routing_key = build_routing_key
+        self.options = options
```

Comment:
Since we're not using `self.options` as we originally thought we can just remove this and `options={}`.

I will do review that the PR works later.